docs/android/v2/features/render-video.mdx

```diff
@@ -78,8 +78,11 @@ hmsVideoView.removeTrack()
 
 Until you call `removeTrack()` the video frame is being streamed over the device's data.
 
-### High Quality Video Frame Capture
-You can get a high quality bitmap object of the frame getting rendered on the HMSVideoView
+### Capture Video Frame
+You can get a bitmap object of the frame getting rendered on the `HMSVideoView`
+
+In case you are calling this API on a `videoView` which is rendering localVideoTrack, it will return a bitmap whose resolution would be equal to the resolution at which this peer is supposed to publish

```

Comment:
```suggestion
In case you are calling this API on a `videoView` which is rendering `localVideoTrack`, it will return a bitmap whose resolution would be equal to the resolution at which this peer is supposed to publish.

```